processor/metricstransformprocessor/README.md

```diff
@@ -17,12 +17,15 @@ The metrics transform processor can be used to rename metrics, labels, or label
   - Aggregation_type: sum, mean, max
 - Add label to an existing metric
 - When adding or updating a label value, specify `{{version}}` to include the application version number
+- Convert resource attributes to metric labels
 
 ## Configuration
 ```yaml
 # transforms is a list of transformations with each element transforming a metric selected by metric name
 transforms:
   # name is used to match with the metric to operate on. This implementation doesn’t utilize the filtermetric’s MatchProperties struct because it doesn’t match well with what I need at this phase. All is needed for this processor at this stage is a single name string that can be used to match with selected metrics. The list of metric names and the match type in the filtermetric’s MatchProperties struct are unnecessary. Also, based on the issue about improving filtering configuration, it seems like this struct is subject to be slightly modified.
+
+  # it also accepts `all_metrics` as the value of 'metric_name' field. This batch operation applies changes to all the metrics. However, currently, it only works and is tested for the 'convert_resource_attributes_to_labels' operation.
```

Comment:
nit: Can you just implement it for all operations in this PR? Or create a separate PR for just the all_metrics change

Reply:
For most of the update operations, it should work fine. I already did some manual testing. Need more unit tests to verify it works fine for the operations we are claiming. Separate PR would make the process faster. Here is the issue I created for tracking- #1342 . Also, for the aggregation operations, I don't know how it will work. That's why planned for a separate PR.